config/config.exs

```diff
@@ -1,26 +1,3 @@
-# This file is responsible for configuring your application
-# and its dependencies with the aid of the Mix.Config module.
```

Comment:
Any reason to remove these two lines of documentation?

Reply:
There are two reasons.

1. This is a misleading message for many new programmers in Elixir land; this does not configure your application if you use this package as a dependency, has been multiple (no on this package) cases where they assume they need to add some default values to these configs.

2. Having those messages add no value to the source code, this is well documented in Elixir for us to have these messages all around the repositories.

In any case, feel free to block the merging, and I will roll the changes back if you think this is not a good idea.